Automattic/wp-calypso
# Editor: Default editor url to block editor

#### Changes proposed in this Pull Request

Now that the Calypso editor is deprecated, default the editor url to the block editor.

We'll remove the `inEditorDeprecationGroup` and related code in separate PRs.

#### Testing instructions

- ~~Sandbox the API and apply D48647-code~~
- Make sure your user is in the editor deprecation group (user attribute `editor_deprecation_group` is true)
- All urls that point to editing a post or page should point to the block editor (`/block-editor/...`)
- Be sure to test various post types: post, pages, jetpack-portfolio, etc.
